sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/filters/TargetingFilter.java

```diff
@@ -171,7 +196,14 @@ public boolean evaluate(FeatureFilterEvaluationContext context) {
 
         String defaultContextId = targetingContext.getUserId() + "\n" + context.getFeatureName();
 
-        return isTargeted(defaultContextId, settings.getAudience().getDefaultRolloutPercentage());
+        return isTargeted(defaultContextId, audience.getDefaultRolloutPercentage());
+    }
+
+    private String getKeyFormat(Map<String, Object> parameters, String key) {
```

Comment:
Maybe `getKeyLowerCase` or `getKeyAsLowerCase` is more straightforward?

Reply:
How about `getKeyCase` as it can return both keys types. i.e. `Users` and `users`